lib/extensions.js

```diff
@@ -89,8 +89,9 @@ function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
          package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
```

Comment:
Yes, I read. Btw. --sass-binary-url didn't work at all, it caused an exception (in current master).